### ISSUE TITLE
WFSS calspec2 errors in create_grism_bbox

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#5239]
_Issue [JP-1640](https://jira.stsci.edu/browse/JP-1640) was created on JIRA by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse):_

The "_create_grism_bbox" function that's called by both the bkg_subtract and extract_2d steps in the calwebb_spec2 pipeline when processing NIRCam and NIRISS WFSS exposures has shown the following error:
```java
File "/dms/local/jwst/pipeline/pkgs/miniconda3/envs/jwstdp-0.16.1.20200718-py37/lib/python3.7/site-packages/jwst/assign_wcs/util.py", line 617, in create_grism_bbox
  return _create_grism_bbox(input_model, mmag_extract, wfss_extract_half_height, wavelength_range)
File "/dms/local/jwst/pipeline/pkgs/miniconda3/envs/jwstdp-0.16.1.20200718-py37/lib/python3.7/site-packages/jwst/assign_wcs/util.py", line 678, in _create_grism_bbox
  xmin = int(np.min(xstack))
ValueError: cannot convert float NaN to integer
```

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1640?focusedCommentId=478863#comment-478863):_

A cursory analysis indicates that this might be caused by sources contained in the input catalog that are slightly outside the field-of-view of the grism exposure that's being processed, because the catalog was created from a set of combined dithered images that cover a slightly large field-of-view on the sky than the individual grism exposures. So the solution could be as simple as ignoring objects that have detector coords outside the limits of the image.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1640?focusedCommentId=479144#comment-479144):_

Fixed by #5241
